assets/css/startertheme.css

```diff
@@ -49,6 +49,21 @@
   vertical-align: middle;
 }
 
+/* ORGANIZATIONS SECTION */
+
+.organizations-section-image {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  float: right;
+  height: 50px;
+  width: 50px;
+}
+
+.max-width-650 {
+  max-width:650px;
```

Comment:
The height, width, and max-width are set to exact values, have you checked what happens on smaller screens? These may not scale nicely.

Reply:
From what I tested, in normal mode and phone mode on chrome, they seemed to be ok. But I'll change the images to rem. 
For the max width, the size I went to, it showed the whole table, and if you go smaller, a scroll bar at the bottom appears. But I can change it if preferred.